src/vars.js

```diff
@@ -505,21 +509,23 @@ exports.node = {
   global        : false,
   module        : false,
   require       : false,
+  Intl          : false,
 
   // These globals are writeable because Node allows the following
   // usage pattern: var Buffer = require("buffer").Buffer;
 
-  Buffer        : true,
-  console       : true,
-  exports       : true,
-  process       : true,
-  setTimeout    : true,
-  clearTimeout  : true,
-  setInterval   : true,
-  clearInterval : true,
-  setImmediate  : true, // v0.9.1+
-  clearImmediate: true, // v0.9.1+
-  URL           : true  // v6.13.0+
+  Buffer         : true,
+  console        : true,
+  exports        : true,
+  process        : true,
+  setTimeout     : true,
+  clearTimeout   : true,
+  setInterval    : true,
+  clearInterval  : true,
+  setImmediate   : true, // v0.9.1+
+  clearImmediate : true, // v0.9.1+
+  URL            : true,  // v6.13.0+
+  URLSearchParams: true  // v6.13.0+
```

Comment:
Not that it matters too much, but URLSearchParams was introduced in v10.0.0+

Reply:
Damn, I really checked the history section of [URLSearchParams](https://nodejs.org/api/url.html#class-urlsearchparams). I saw it was **introduced** in _v6.13.0_, but didn't notice that it became a **global variable** only in _v10.0.0_.

Reply:
@coolaj86 Accurate documentation matters plenty! Thanks for checking our work :) Fixed via commit 06accfab74385e33ece831dc877cbafb20d9b515